core/src/View/Kokkos_ViewCtor.hpp

```diff
@@ -236,11 +235,12 @@ struct ViewCtorProp : public ViewCtorProp<void, P>... {
   template <typename... Args>
   ViewCtorProp(view_ctor_prop_args<Args...> const &arg)
       : view_ctor_prop_base<Args>(
-            static_cast<view_ctor_prop_base<Args> const &>(arg))... {
-    // Suppress an unused argument warning that (at least at one point) would
-    // show up if sizeof...(Args) == 0
-    (void)arg;
```

Comment:
Just a note for the reviewers: it looks like this is no longer needed after adding the default constructor.

Reply:
Reverted.